derive_client/utils/w3.py

```diff
@@ -127,10 +126,6 @@ def send_and_confirm_tx(
         print(f"⏱️ Timeout waiting for tx receipt of {tx_hash.hex()}")
         tx_result.exception = timeout_err
         return tx_result
-    except Exception as wait_err:
-        print(f"⚠️ Error while waiting for tx receipt of {tx_hash.hex()}: {wait_err!r}")
-        tx_result.exception = wait_err
-        return tx_result
```

Comment:
`wait_for_tx_receipt` can only result in the `tx_receipt` or a `TimeoutError`. This branch is logically impossible to hit